### DISCUSSION TITLE
Dropzone image not found and weird slash

### DISCUSSION THREAD

**Discussion by Tuoww** [Laravel-Backpack/community-forum#578] [Q&A (Help)] [answered]
Hi, when I want to upload some files via Dropzone using: 
`CRUD::field('photos')->type('dropzone')->withFiles(['path' => 'productphotos']);` it uploads them in the correct folder.

But when I then want to show the link/url in the list view with: 
`CRUD::column('photos')->type('dropzone');`
The link is: /storage/productphotos//plaatje1-koHd.jpg and this image is not found. And the double / between productphotos and name of the image/file is a bit weird.

So I was wondering if anyone else already had this problem and if so how they fixed it.
Thanks in advance

This is my filesystems.php
```
`'local' => [
            'driver' => 'local',
            'root' => storage_path('app'),
            'throw' => false,
        ],

        'public' => [
            'driver' => 'local',
            'root' => storage_path('app/public'),
            'url' => env('APP_URL').'/storage',
            'visibility' => 'public',
            'throw' => false,
        ],`

```

**Comment by pxpm** [comment]
Hello @Tuoww you also need to add the `->withFiles()` definition to your column definition. 

Let me know how it goes.

Cheers

**Reply by Tuoww** [reply]
I changed it to: `CRUD::column('photos')->type('dropzone')->withFiles();` now the label has the right value I think: `productphotos/plaatje1-koHd.jpg` but the href still is `/storage/productphotos//plaatje1-koHd.jpg` and still not found.

When I add `->withFiles(['path' => 'productphotos'])` only the label changes to `productphotosproductphotos/plaatje1-koHd.jpg`

**Reply by pxpm** [reply]
I will investigate and get back to you. 

Thanks. 🙏

**Reply by Tuoww** [reply]
The label is also still wrong = 'productphotosproductphotos//plaatje1-koHd.jpg' with double backslashes

**Reply by pxpm** [reply]
I found the issues. Already submitted the PR to PRO to fix them. 

My colleague will review and we will merge shortly, I will ping you when you can `composer update` to get the fix.

Thanks again 🙏 

Cheers

**Reply by Tuoww** [reply]
And is that a fix for the double / part or for the not found part.
Anyway thanks for your quick response!

**Reply by pxpm** [reply]
For both of them. 
Everything should be good now @Tuoww 🙏 

If not, plese open other issue! 

Thanks

**Comment by pxpm** [comment, marked as answer]
This had been fixed in PRO 2.0.8. 

Cheers
